genie-web/src/test/resources/com/netflix/genie/web/controllers/JobRestControllerIntegrationTests/linux-runsh.txt

```diff
@@ -7,12 +7,14 @@ trap "handle_failure" ERR EXIT
 
 function handle_failure {
     ERROR_CODE=$?
+    echo "Handling exit signal (code: ${ERROR_CODE})"
+
     # Good exit
     if [[ ${ERROR_CODE} -eq 0 ]]; then
         exit 0
     fi
     # Bad exit
-    printf '{"exitCode": "%s"}\n' "${ERROR_CODE}" > ./genie/genie.done
+    printf '{"exitCode": "%s"}\n' "${ERROR_CODE}" > ${GENIE_JOB_DIR}/genie/genie.done
```

Comment:
This (and the next two hardcoded comments) are on expected outputs that we compare against in test. I don't think we should remove this string (if we do, we need to export $GENIE_DONE_FILE as environment variable in the run script.